src/value.jl

```diff
@@ -140,8 +153,11 @@ function Base.isless(x::CategoricalValue, y::CategoricalValue)
     end
 end
 
-Base.isless(x::CategoricalValue, y::SupportedTypes) = levelcode(x) < levelcode(x.pool[get(x.pool, y)])
-Base.isless(y::SupportedTypes, x::CategoricalValue) = levelcode(x.pool[get(x.pool, y)]) < levelcode(x)
+Base.isless(x::CategoricalValue, y::SupportedTypes) =
```

Comment:
so you have decided to go forward fast and be breaking here? (I am OK with this, but usually you wanted one round of releases with deprecations :smile: )

Reply:
Yes maybe I can backport a deprecation to 0.9. Though the error message is basically the same as a deprecation so I wonder whether it's worth it.

Reply:
I am OK with error as commented.